DOCUMENTATION.md

```diff
@@ -279,6 +279,18 @@ packages you have access to. 
 content-cli list packages -p <your-chosen-profile>
 ```
 
+### List all assets in Studio 
+
+With this command you can retrieve a list of all assets with the --assetType option within a team.
+The command takes your permissions into consideration and only lists the
+assets you have access to. 
+
+-   It is also possible to download assets in YML format by adding '--yml' option.
```

Comment:
Why are we doing YAML now, can we just keep it in JSON like the other responses?

Reply:
@promeris seems yml responses have been used for the assets. The writing file base function uses [this overwritten method](https://github.com/celonis/content-cli/blob/b4d34703173838f19d622405cd760ed63b5ad4dd/src/content/manager/asset.manager.ts#L123). Wanted to stick with it. Otherwise there will be a difference between pull & list assets cmds response types.